File: app/src/main/java/kr/kro/awesometic/plankhelper/util/LogManager.java
package kr.kro.awesometic.plankhelper.util;

import android.util.Log;

/**
 * Created by Awesometic on 2017-06-12.
 */

public class LogManager {

    public static void v(Object content) {
        if (content instanceof String) {
            Log.d(Constants.LOG_TAG, (String) content);
        } else {
            Log.d(Constants.LOG_TAG, String.valueOf(content));
        }
    }

    public static void d(Object content) {
        if (content instanceof String) {
            Log.d(Constants.LOG_TAG, (String) content);
        } else {
            Log.d(Constants.LOG_TAG, String.valueOf(content));   
        }
    }

    public static void i(Object content) {
        if (content instanceof String) {
            Log.d(Constants.LOG_TAG, (String) content);
        } else {
            Log.d(Constants.LOG_TAG, String.valueOf(content));
        }
    }

    public static void w(Object content) {
        if (content instanceof String) {
            Log.d(Constants.LOG_TAG, (String) content);
        } else {
            Log.d(Constants.LOG_TAG, String.valueOf(content));
        }
    }

    public static void e(Object content) {
        if (content instanceof String) {
            Log.d(Constants.LOG_TAG, (String) content);
        } else {
            Log.d(Constants.LOG_TAG, String.valueOf(content));
        }
    }
}

File: app/src/main/java/kr/kro/awesometic/plankhelper/util/LogManager.kt
package kr.kro.awesometic.plankhelper.util

import android.util.Log

/**
 * Created by Awesometic on 2017-06-12.
 */

object LogManager {

    fun v(content: Any) {
        if (content is String) {
            Log.d(Constants.LOG_TAG, content)
        } else {
            Log.d(Constants.LOG_TAG, content.toString())
        }
    }

    fun d(content: Any) {
        if (content is String) {
            Log.d(Constants.LOG_TAG, content)
        } else {
            Log.d(Constants.LOG_TAG, content.toString())
        }
    }

    fun i(content: Any) {
        if (content is String) {
            Log.d(Constants.LOG_TAG, content)
        } else {
            Log.d(Constants.LOG_TAG, content.toString())
        }
    }

    fun w(content: Any) {
        if (content is String) {
            Log.d(Constants.LOG_TAG, content)
        } else {
            Log.d(Constants.LOG_TAG, content.toString())
        }
    }

    fun e(content: Any) {
        if (content is String) {
            Log.d(Constants.LOG_TAG, content)
        } else {
            Log.d(Constants.LOG_TAG, content.toString())
        }
    }
}
